Sources/Commands/SwiftPackageTool.swift

```diff
@@ -158,6 +158,7 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
                     version: resolveOptions.version.flatMap(Version.init(string:)),
                     branch: resolveOptions.branch,
                     revision: resolveOptions.revision,
+                    shouldResolveMissingDepenencies: true,
```

Comment:
Just FYI, there's a typo here.

```suggestion
                    shouldResolveMissingDependencies: true,
```